trieste/acquisition/combination.py

```diff
@@ -26,24 +26,21 @@
 
 class Reducer(AcquisitionFunctionBuilder):
     r"""
-    A :class:`Reducer` builds an :func:`~trieste.acquisition.AcquisitionFunction` whose output is
+    A :class:`Reducer` builds an :data:`~trieste.acquisition.AcquisitionFunction` whose output is
```

Comment:
Is `:data:` intentional?

Reply:
it is: `:func:` is for actual functions, `:data:` is the best option I could find for type aliases. I think it's a generic "this is a fixed module-level thing".

Reply:
`:const:` was another option, can't recall why I chose `:data:`

Reply:
oh, actually `:data:` is for module-level variables, so I'll use `:const:`